internal/version/test/main_test.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/tetratelabs/wazero/internal/testing/require"
+	"github.com/tetratelabs/wazero/internal/version"
+)
+
+// TestGetWazeroVersion ensures that GetWazeroVersion returns the version of wazero in the go.mod in the
+// downstream wazero users.
+func TestGetWazeroVersion(t *testing.T) {
+	// This matches the one in the "replace" statement in the go.mod.
+	const exp = "v0.0.0-20220818123113-1948909ec0b1"
```

Comment:
looks too much like a real version, maybe set it to v0.0.0? we are wazero after all

Reply:
this must be a real one otherwise go test ./... will fail saying that wazero not found regardless of the existence of replace statement in go.mod